streams/src/test/java/org/apache/kafka/streams/processor/internals/assignment/LegacySubscriptionInfoSerde.java

```diff
@@ -41,13 +45,15 @@
     private final Set<TaskId> prevTasks;
     private final Set<TaskId> standbyTasks;
     private final String userEndPoint;
+    private final Map<TaskId, Integer> taskLags;
 
     public LegacySubscriptionInfoSerde(final int version,
                                        final int latestSupportedVersion,
                                        final UUID processId,
                                        final Set<TaskId> prevTasks,
                                        final Set<TaskId> standbyTasks,
-                                       final String userEndPoint) {
+                                       final String userEndPoint,
+                                       final Map<TaskId, Integer> taskLags) {
```

Comment:
I'm probably missing the point here, but I think the idea of this class is that it should not change in response to changes in SubscriptionInfo. I think it's supposed to be a stand-in for the behavior of older Streams versions when the cluster has old and new members running at the same time. Maybe it doesn't really work that way, though, in which case, I might doubt the utility of this class at all, and instead recommend relying on the system tests. Can you comment?